web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -12,31 +12,45 @@ import { Certification, Funding, SidebarCardContent } from "@/lib/types/types";
 import analytics from "@/lib/utils/analytics";
 import { openInNewTab, scrollToTopOfElement, templateEval } from "@/lib/utils/helpers";
 import { isRemoteWorkerOrSellerBusiness } from "@businessnjgovnavigator/shared/domain-logic/businessPersonaHelpers";
+import { LookupOperatingPhaseById } from "@businessnjgovnavigator/shared/operatingPhase";
 import { Accordion, AccordionDetails, AccordionSummary, useMediaQuery } from "@mui/material";
 import { ReactElement, ReactNode, useRef, useState } from "react";
 
 export interface SidebarCardsListProps {
-  topCards: SidebarCardContent[];
-  bottomCards: SidebarCardContent[];
+  sideBarCards: SidebarCardContent[];
   fundings: Funding[];
   hiddenFundings: Funding[];
   certifications: Certification[];
   hiddenCertifications: Certification[];
-  displayFundings: boolean;
-  displayCertifications: boolean;
-  isCMSPreview?: boolean;
+  CMS_ONLY_DISPLAY_FUNDINGS?: boolean;
+  CMS_ONLY_DISPLAY_CERTIFICATIONS?: boolean;
+  CMS_ONLY_IS_REMOTE_SELLER_WORKER?: boolean;
 }
 
 export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement => {
   const [hiddenAccordionIsOpen, setHiddenAccordionIsOpen] = useState<boolean>(false);
   const { Config } = useConfig();
   const { business } = useUserData();
   const accordionRef = useRef(null);
+  const isDesktopAndUp = useMediaQuery(MediaQueries.desktopAndUp);
+
+  const displayFundingCards = (): boolean => {
+    return (
+      props.CMS_ONLY_DISPLAY_FUNDINGS ||
+      LookupOperatingPhaseById(business?.profileData.operatingPhase).displayFundings
+    );
+  };
+  const displayCertificationsCards = (): boolean => {
+    return (
+      props.CMS_ONLY_DISPLAY_CERTIFICATIONS ||
+      LookupOperatingPhaseById(business?.profileData.operatingPhase).displayCertifications
+    );
+  };
```

Comment:
[dust] super nit picky of me, but I probably would have just calculated the value instead of making arrow functions.

```suggestion
  const displayFundingCards = props.CMS_ONLY_DISPLAY_FUNDINGS || LookupOperatingPhaseById(business?.profileData.operatingPhase).displayFundings;
  
  const displayCertificationsCards = props.CMS_ONLY_DISPLAY_CERTIFICATIONS || LookupOperatingPhaseById(business?.profileData.operatingPhase).displayCertifications;
```

Reply:
yeah, I missed this, I will update, Spencer brought it up as well.